### ISSUE TITLE
need support for mongoose 6

### ISSUE THREAD

**Issue by zhaydexxa** [vikpe/mongoose-mpath#37]
hi there,

after upgrade mongoose from 5 to 6, parent populate not working anymore.

https://mongoosejs.com/docs/migrating_to_6.html
No default model for Query.prototype.populate

**Comment by vikpe** [comment]
@zhaydexxa I've updated mongoose to 6.x in release 3.0, could you try it?

(sorry for a very late update)

**Comment by zhaydexxa** [comment]
still not working

i have to add a model in populate query to make it run properly in mongoose 6
`await TestModel.find({}).populate([{ path: 'parent', model: TestModel }]);`

**Comment by vikpe** [comment]
thank you for feedback. 

since mongoose 6.x, [populate](https://mongoosejs.com/docs/populate.html) requires `ref` to be set (name of the model).

I've published `mongoose-mpath 3.0.1` where option `modelName` now is required; e.g: 
```js
const LocationSchema = new Mongoose.Schema({name: String});
LocationSchema.plugin(MpathPlugin, {modelName: 'Location'});

const LocationModel = Mongoose.model('Location', LocationSchema);
```

now populate should work, e.g:
```js
const sthlm = await Location.findOne({ name: 'Stockholm' }).populate('parent');
console.log(sthlm.parent.name); // Sweden
```
([see test case](https://github.com/vikpe/mongoose-mpath/blob/master/test/mpath-test.js#L1000))

**Comment by zhaydexxa** [comment]
ok it's work ..
thanks
